vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ExhibitionLanguage.java

```diff
@@ -26,6 +33,9 @@ public class ExhibitionLanguage extends VSpaceElement implements IExhibitionLang
     
     private boolean isDefault;
     
+    @OneToMany(targetEntity = LocalizedText.class ,  cascade= CascadeType.ALL, orphanRemoval = true)    
```

Comment:
I don't think we want that. If an exhibition language is accidentally deleted, we don't want to delete all the texts as well! That might have been hours of work to enter them.